HTTPServerProject.Tests/HTTPServerWriteTests/TestWriteStreams.cs

```diff
@@ -0,0 +1,64 @@
+using HTTPServerWrite.Streams;
+
+namespace HTTPServerWriteTests.Streams
```

Comment:
This is inconsistent with the way you've been defining namespaces everywhere else.

Reply:
Will update.